scripts/biomedical/NCBI_Taxonomy/test_data/division_enum_expected.mcf

```diff
@@ -0,0 +1,78 @@
+Node:dcid:BiologicalTaxonomicDivisionEnum
+name: "BiologicalTaxonomicDivisionEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "These are broad biological divisions that are commonly used to divide living organisms"
+
+Node:dcid:BiologicalTaxonomicDivisionBacteria
+name: Bacteria
```

Comment:
All text value for properties should be in "".  This includes name, acronym, and description (e.g. name: "Bacteria"). Please update this in your script!